app.js

```diff
@@ -29,13 +31,26 @@ function showSlide(n) {
 function counter() {
 	for (let i = 0; i < number.length; i++) {
 		number[i].innerHTML = slideindex + "/3";
+		img[i].src = "css/image/img" + slideindex + ".jpg";

```

Comment:
this logic will work only if images have a name with only digits and `jpg` format.